packages/atomic/src/components/search/tabs/atomic-tab-manager/atomic-tab-manager.pcss

```diff
@@ -1,11 +1,13 @@
 @import '../../../../global/global.pcss';
 
-@media (max-width: 768px) {
-  .tab-dropdown-area {
-    display: block;
-  }
+.hide-tabs {
+  height: 0;
+  visibility: hidden;
+  margin: 0;
+  padding: 0;
+}
 
-  .tab-area {
-    display: none;
-  }
+select:hover + div,
+select:focus-visible + div {
+  @apply text-primary-light;
```

Comment:
The box shadow makes it a bit weird:

<img width="504" alt="Screenshot 2024-10-08 at 10 59 25 AM" src="https://github.com/user-attachments/assets/8d34ec77-6571-4380-bd7f-8ce981582229">

Reply:
This is exactly the same css as in the sort criteria dropdown so I'll have to check what causes this glitch here

Reply:
This occurs because the select element takes the full width of its container, I can add some padding to the container to avoid the issue :)